src/panels/config/zwave/zwave-network.ts

```diff
@@ -42,157 +57,197 @@ export class ZwaveNetwork extends LitElement {
-            >
-            </ha-service-description>
-
-            <ha-call-service-button
-              .hass=${this.hass}
-              domain="zwave"
-              service="stop_network"
-            >
-              Stop Network
-            </ha-call-service-button>
-            <ha-service-description
-              .hass=${this.hass}
-              domain="zwave"
-              service="stop_network"
-              ?hidden=${!this._showHelp}
-            >
-            </ha-service-description>
-
-            <ha-call-service-button
-              .hass=${this.hass}
-              domain="zwave"
-              service="soft_reset"
-            >
-              Soft Reset
-            </ha-call-service-button>
-            <ha-service-description
-              .hass=${this.hass}
-              domain="zwave"
-              service="soft_reset"
-              ?hidden=${!this._showHelp}
-            >
-            </ha-service-description>
-
-            <ha-call-service-button
-              .hass=${this.hass}
-              domain="zwave"
-              service="test_network"
-            >
-              Test Network
-            </ha-call-service-button>
-            <ha-service-description
-              .hass=${this.hass}
-              domain="zwave"
-              service="test_network"
-              ?hidden=${!this._showHelp}
-            >
-            </ha-service-description>
-
-            <ha-call-api-button .hass=${this.hass} path="zwave/saveconfig">
-              Save Config
-            </ha-call-api-button>
-          </div>
-        </ha-card>
+        ${this._networkStatus
+          ? html`
+              <ha-card class="content network-status">
+                <div class="details">
+                  ${this._networkStarting
+                    ? html`
+                        <paper-spinner
+                          active
+                          alt="Starting Network..."
+                        ></paper-spinner>
+                        Starting Z-Wave Network...<br />
+                        <small
+                          >This may take a while depending on the size of your
+                          network.</small
+                        >
+                      `
+                    : html`
+                        ${this._networkStatus.state === 0
+                          ? html`
+                              <ha-icon icon="hass:close"> </ha-icon>
+                              Z-Wave Network Stopped
+                            `
+                          : ""}
+                        ${this._networkStatus.state === 5
+                          ? html`
+                              <paper-spinner
+                                alt="Starting Network..."
+                              ></paper-spinner>
+                              Z-Wave Network Starting
+                            `
+                          : ""}
+                        ${this._networkStatus.state === 7
```

Comment:
can we create `export const ZWAVE_NETWORK_STATE_STARTED` constants in `data/zwave`